backend/api/Database/Models/InspectionFindings.cs

```diff
@@ -1,22 +1,42 @@
-﻿using Microsoft.EntityFrameworkCore;
-
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Api.Controllers.Models;
 #pragma warning disable CS8618
 namespace Api.Database.Models
 {
-    [Owned]
     public class InspectionFindings
     {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public string Id { get; set; }
 
         public string RobotName { get; set; }
```

Comment:
Not sure we need a string RobotName here. The inspection, task, missionRun already has information about which robot ran the mission I think.

Reply:
Done.